src/CMakeLists.txt

```diff
@@ -65,7 +65,8 @@ set(HEADER_FILES
   ${LIBCELLML_CONFIG_H}
 )
 
-add_compiler_export_flags()
+set(CMAKE_CXX_VISIBILITY_PRESET hidden)
+set(CMAKE_VISIBILITY_INLINES_HIDDEN 1)
```

Comment:
Good point. I believe this should now be the case with https://github.com/cellml/libcellml/pull/73/commits/d7aaf9346f7952691709be0aa8571637314f0e59.